src/pages/api/chooseValidity.ts

```diff
@@ -1,13 +1,11 @@
 import { NextApiResponse } from 'next';
 import { NextApiRequestWithSession, ErrorInfo } from '../../interfaces/index';
 import { updateSessionAttribute } from '../../utils/sessions';
-import { DAYS_VALID_ATTRIBUTE } from '../../constants/index';
+import { TIME_PERIOD_VALID_ATTRIBUTE } from '../../constants/index';
 import { redirectToError, redirectTo } from './apiUtils';
 import { isSessionValid } from './apiUtils/validator';
 
-export const isInvalidValidityNumber = (req: NextApiRequestWithSession): boolean => {
-    const { validityInput } = req.body;
-
+export const isInvalidValidityNumber = (validityInput: number): boolean => {
```

Comment:
While we're making a change in here, could we update the name of this function? I have no idea what its doing from just looking at its name

Reply:
Really? The name returns a boolean, and its called isInvalidValidityNumber - so its checking if the Validity Number is valid?

Reply:
Maybe its just the way I read it, but its not immediately clear to me. Happy to leave it be though if you think otherwise 🙂